templates/github/.ci/ansible/start_container.yaml

```diff
@@ -81,13 +81,26 @@
           command: "docker logs pulp"
           failed_when: true
 
-    - name: "Check version of component being tested"
-      assert:
-        that:
-          - (result.json.versions | items2dict(key_name="component", value_name="version"))[component_name] | canonical_semver == (component_version | canonical_semver)
-        fail_msg: |
-          Component {{ component_name }} was expected to be installed in version {{ component_version }}.
-          Instead it is reported as version {{ (result.json.versions | items2dict(key_name="component", value_name="version"))[component_name] }}.
+    - block:
+        - name: "Check the component is in status"
+          assert:
+            that:
+              - component_name in (result.json.versions | items2dict(key_name="component", value_name="version") | keys)
```

Comment:
So the keys filter does not exist... still the rescue block is executed.
Then we should be fine just removing this task.

Reply:
I was so laser focused seeing a failure, that i saw a failure, not realizing it was the wrong one.